extensions/skins/conversation_v1/Conversation.js

```diff
@@ -589,7 +588,7 @@ oppia.directive('conversationSkin', [function() {
       });
 
       $window.addEventListener('beforeunload', function(e) {
-        if (hasInteractedAtLeastOnce && !$scope.explorationCompleted &&
+        if (hasInteractedAtLeastOnce && !$scope.isOnFinalCard() &&
```

Comment:
I posted the change but did so under a different username by accident.  I'm not sure how to remedy this. Does it matter?